### PR TITLE
Feature hero or

### PR DESCRIPTION
**Pull request by OleksandrRyzhov2299**

### CODE REVIEW

File: src/partials/hero.html [hunk 1]
```diff
@@ -1,5 +1,15 @@
 <section class="hero">
   <div class="container">
-    <h1>Discover the joy of learning English with us!</h1>
+    <div class="hero-wrapper">
+      <h1 class="hero-title">
+        Discover the joy of learning <span>English</span> with us!
+      </h1>
+      <a class="hero-link" href="#">Leave an application</a>
```
Comment (VOSolyanik) [inline src/partials/hero.html]: hero-link -> btn btn-primary

Comment (VOSolyanik) [inline src/partials/hero.html]: href="#leave-application"

File: src/css/sections/hero.css [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    height: 48px;
+  }
+}
+@media screen and (min-width: 1280px) {
+  .hero-scroll-down {
+    margin-top: 83px;
+  }
+}
+.hero-icon {
+  fill: currentColor;
```
Comment (VOSolyanik) [inline src/css/sections/hero.css]: icon is not visible

File: src/css/sections/hero.css [hunk 3]
```diff
@@ -0,0 +1,177 @@
+.hero-wrapper {
```
Comment (VOSolyanik) [inline src/css/sections/hero.css:6]: hero-wrapper on desktop should be 1152px but now 1024

Comment (VOSolyanik) [reply src/css/sections/hero.css:6]: use negative margin or set `.hero .container { paddong: 0 }`

Comment (VOSolyanik) [reply src/css/sections/hero.css:6]: also please add for .header

.header { margin-bottom: 8px)

min-width: 768px 
    .header { margin-bottom: 32px)
    
min-width: 1152px 
    .header { margin-bottom: 16px)

File: src/images/hero/bg_desk@2x.jpg
Comment (VOSolyanik) [inline src/images/hero/bg_desk@2x.jpg:1]: Images is not optimized

Screenshot 2024-10-04 at 00 57 21
Screenshot 2024-10-04 at 00 57 14

**Review by VOSolyanik — CHANGES_REQUESTED** [review]

File: src/partials/hero.html (hunk 1 above)
Comment (yuratouch) [reply src/partials/hero.html]: fixed

Comment (yuratouch) [reply src/partials/hero.html]: fixed

File: src/css/sections/hero.css (hunk 2 above)
Comment (yuratouch) [reply src/css/sections/hero.css]: fixed

File: src/css/sections/hero.css (hunk 3 above)
Comment (yuratouch) [reply src/css/sections/hero.css:6]: all fixed

File: src/images/hero/bg_desk@2x.jpg
Comment (yuratouch) [reply src/images/hero/bg_desk@2x.jpg:1]: images optimized. reduced 74% for jpg and 5% for webp

**Comment by yuratouch** [comment]
[HC-12. Hero Section](https://trello.com/c/aIXZwIup/7-hc-12-hero-section)

**Review by VOSolyanik — APPROVED** [review]

**Pull request merged**